frontend/src/component/integrations/CreateIntegration/CreateIntegration.tsx

```diff
@@ -14,7 +14,7 @@ export const DEFAULT_DATA = {
     environments: [],
 } as unknown as IAddon; // TODO: improve type
 
-export const CreateAddon = () => {
+export const CreateIntegration = () => {
     const providerId = useRequiredPathParam('providerId');
     const { providers, refetchAddons } = useAddons();
```

Comment:
Should we update the hook also so it is consistently named, to integration?

Reply:
Everything on backend is still called _Addon_. I suggest leaving it like this, otherwise we need to refactor backend too. Reason for renaming frontend files is it will be easier to find it if files are named in the same way as menu item